{"id":27229,"className":"javax.xml.datatype.DatatypeConfigurationException","methodName":"DatatypeConfigurationException","signature":"\u003cjavax.xml.datatype.DatatypeConfigurationException: void DatatypeConfigurationException(java.lang.Throwable)\u003e","javadoc":"/** \n * \u003cp\u003eCreate a new \u003ccode\u003eDatatypeConfigurationException\u003c/code\u003e with the specified cause.\u003c/p\u003e\n * @param cause The cause.  A \u003ccode\u003enull\u003c/code\u003e value is permitted, and indicates that the cause is nonexistent or unknown.\n */\n","tf":{"freq":{"except":2,"configur":2,"caus":4,"indic":1,"nonexist":1,"creat":1,"permit":1,"valu":1,"unknown":1,"datatyp":2,"specifi":1},"maxFreq":4,"totalTerms":17},"loc":3}